def/flow.js

```diff
@@ -287,4 +287,9 @@ module.exports = function (fork) {
         def("Literal"),
         null
       ), defaults["null"]);
+
+    def("DeclareModuleExports")
+      .bases("Statement")
```

Comment:
I'm not sure. That's copied directly from Flow's source; also, the surface
syntax is `declare module.exports = foo` which looks kind of like an
assignment statement... :smile:

On Aug 6, 2016 6:50 PM, "Ben Newman" <notifications@github.com> wrote:

> In def/flow.js
> <https://github.com/benjamn/ast-types/pull/169#discussion_r73787916>:
>
> > @@ -287,4 +287,9 @@ module.exports = function (fork) {
> >          def("Literal"),
> >          null
> >        ), defaults["null"]);
> > +
> > +    def("DeclareModuleExports")
> > +      .bases("Statement")
>
> Not .bases("Declaration")?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/benjamn/ast-types/pull/169/files/e7a7713487cfd8959f9056eb6535e3a7e80bd937#r73787916>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACJHpZUUgszy5Uil9__4-Q-mgtlTf8cZks5qdK1HgaJpZM4JeO0b>
> .
>